_posts/2015-03-24-RCHomework.Rmd

```diff
@@ -3,6 +3,7 @@ title: "Blog Post"
 author: "Richie Castellanes"
 date: "March 22, 2015"
 output: html_document
+published: false
```

Comment:
jekyll will use the rendered html instead of trying to recompile this one. Only the other one will show up in the archive index page